### PR TITLE
Make dark mode readable for subtests status

### PR DESCRIPTION
**Pull request by Netacci**
[Bugzilla link](https://bugzilla.mozilla.org/show_bug.cgi?id=1932430)
[Local test link for regression status](http://localhost:3000/subtests-compare-results?baseRev=94366b71b2018d30e08baa7db5d6d87951df13fd&baseRepo=mozilla-central&newRev=a1dd09632bac76382f7edd7187e6f5ee3e12ee68&newRepo=mozilla-central&framework=1&baseParentSignature=86574&newParentSignature=86574)
[Local test link for improvement status](http://localhost:3000/subtests-compare-results?baseRev=0b6cd0f0e624eb2d9ba717594cf9ca625bfcd395&baseRepo=try&newRev=2cebc60940117762bd9c8a221f6f9be6b96cd508&newRepo=try&framework=13&baseParentSignature=223447&newParentSignature=223447)

[Deploy test link for regression status](https://deploy-preview-817--mozilla-perfcompare.netlify.app/subtests-compare-results?baseRev=94366b71b2018d30e08baa7db5d6d87951df13fd&baseRepo=mozilla-central&newRev=a1dd09632bac76382f7edd7187e6f5ee3e12ee68&newRepo=mozilla-central&framework=1&baseParentSignature=86574&newParentSignature=86574)
[Deploy test link for improvement status](https://deploy-preview-817--mozilla-perfcompare.netlify.app/subtests-compare-results?baseRev=0b6cd0f0e624eb2d9ba717594cf9ca625bfcd395&baseRepo=try&newRev=2cebc60940117762bd9c8a221f6f9be6b96cd508&newRepo=try&framework=13&baseParentSignature=223447&newParentSignature=223447)

Before
Dark mode text for improvement and regression status is hard to read for Subtest comparison 

Before
before-regression

After
- Add background colors to `protocolTheme` for uniformity
- Change the `span` element holding the icons to an MUI `Box` and remove all color and bgColor stylings for icons for both `SubtestsRevisionRow` and `RevisionRow`, then conditionally render it on the `SubtestsRevisionRow` and `RevisionRow` components. Also, making sure to set bgColor to none when improvement and regression status are not being displayed so it doesn't affect the 'no changes' status

after

after-regression

### CODE REVIEW

File: src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx [hunk 1]
```diff
@@ -77,21 +77,22 @@ function getStyles(themeMode: string) {
           padding: '4px 10px',
         },
         '.status-hint-improvement': {
-          backgroundColor: '#D8EEDC',
+          backgroundColor: '#004725',
+          marginTop: '2px',
         },
         '.status-hint-regression': {
-          backgroundColor: '#FFE8E8',
+          backgroundColor: '#690F22',
```
Comment (Carla-Moz) [inline src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx]: Moving forward, we want to use MUI theme to update the styles. So, for this PR could you please delete the backgroundColor styles for both improvement and regression  and add the `color` prop to the icons and have it equal to `success`

```
 {improvement ? <ThumbUpIcon color='success' /> : null}
{regression ? <ThumbDownIcon color='success' /> : null}
```

**Review by Carla-Moz — CHANGES_REQUESTED** [review]
Thank you for your PR! I have some suggestions below

File: src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx (hunk 1 above)
Comment (Netacci) [reply src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx]: Should this also be done for `RevisionRows` or only for SubtestsRevisionRow?

Comment (julienw) [reply src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx]: you can do it for both

Comment (julienw) [reply src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx]: I believe carla wanted to say `error` for the `regression` line.

@Carla-Moz I think this works for the icons but I don't think we have anything for the background color yet. I believe we need to add `success` and `error` properties to the theme in https://github.com/mozilla/perfcompare/blob/main/src/theme/protocolTheme.ts#L12 and https://github.com/mozilla/perfcompare/blob/main/src/theme/protocolTheme.ts#L34 ?

Comment (Netacci) [reply src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx]: > you can do it for both

Okay noted

Comment (csevere) [reply src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx]: > I believe carla wanted to say `error` for the `regression` line.
> 
> 
> 
> @Carla-Moz I think this works for the icons but I don't think we have anything for the background color yet. I believe we need to add `success` and `error` properties to the theme in https://github.com/mozilla/perfcompare/blob/main/src/theme/protocolTheme.ts#L12 and https://github.com/mozilla/perfcompare/blob/main/src/theme/protocolTheme.ts#L34 ?

Yes that's why I told Neta to add the new properties to the protocolTheme file in the next commit. Please check my comment further below.

**Review by Carla-Moz — APPROVED** [review]
This looks good to me now. Thanks!

**Pull request merged**